### ISSUE TITLE
Hunter's Prelude Gives Negative Racc

### ISSUE THREAD

**Issue by Aeshur** [EdenServer/community#519]
Unsure if the other prelude is giving accurate amount of +racc but using /checkparam <me> I saw the lower tier prelude, Hunter's Prelude gave me -8 racc at level 31 sync.

**Comment by tectal831** [comment]
/checkparam is disabled on this server, how were you able to use it?

**Comment by godmode** [comment]
It is not disabled afaik, It was disabled on Kupo.

**Comment by tectal831** [comment]
just did /checkparam in game now and it isnt available to use.

**Comment by godmode** [comment]
I was unable to reproduce this. Works fine on test server.

**Comment by Alamandaros** [comment]
![image](https://user-images.githubusercontent.com/46425313/51074886-0009fb80-165b-11e9-9c4c-b1f93fff1af7.png)

> Casting Hunter's Prelude will lower RACC instead of increase. Confirmed by multiple other people in my party. I've been told this is a DSP related bug and only happens at lower levels. I'm unsure as to what level it starts giving proper RACC.

My job was a 34 BRD/WHM (the image is from myself). The two other people in my group when I tested this was a 30 RNG/NIN and a 33 WHM/BLM, both of whom confirmed the same thing.

Checked on a 62 SAM / 25 WHM and it lowered their RACC from 108 to 100.

Checked again on a 75 THF / 37 NIN and it lowered their RACC from 270 to 262.
